qiskit/providers/fake_provider/fake_generic.py

```diff
@@ -0,0 +1,546 @@
+        super().__init__(
+            provider=None,
+            name=f"generic_backend_{num_qubits}q",
+            description=f"This is a device with {num_qubits} qubits and generic settings.",
+            backend_version="",
+        )
+
+        self._sim = None
+        self._rng = np.random.default_rng(seed=seed)
+        self._dtm = dtm
+        self._num_qubits = num_qubits
+        self._control_flow = control_flow
+        self._calibrate_instructions = calibrate_instructions
+        self._supported_gates = get_standard_gate_name_mapping()
+
+        if coupling_map is None:
+            self._coupling_map = CouplingMap().from_full(num_qubits)
+        else:
+            if isinstance(coupling_map, CouplingMap):
+                self._coupling_map = coupling_map
+            else:
+                self._coupling_map = CouplingMap(coupling_map)
+
+            if num_qubits != self._coupling_map.size():
+                raise QiskitError(
+                    f"The number of qubits (got {num_qubits}) must match "
+                    f"the size of the provided coupling map (got {coupling_map.size()})."
+                )
+
+        self._basis_gates = (
+            basis_gates if basis_gates is not None else ["cx", "id", "rz", "sx", "x"]
+        )
+        for name in ["reset", "delay", "measure"]:
+            if name not in self._basis_gates:
+                self._basis_gates.append(name)
+
+        self._build_generic_target()
+        self._build_default_channels()
+
+    @property
+    def target(self):
+        return self._target
+
+    @property
+    def max_circuits(self):
+        return None
+
+    @property
+    def dtm(self) -> float:
+        """Return the system time resolution of output signals"""
+        # converting `dtm` from nanoseconds to seconds
+        return self._dtm * 1e-9 if self._dtm is not None else None
+
+    @property
+    def meas_map(self) -> list[list[int]]:
+        return self._target.concurrent_measurements
+
+    def _build_default_channels(self) -> None:
+        channels_map = {
+            "acquire": {(i,): [pulse.AcquireChannel(i)] for i in range(self.num_qubits)},
+            "drive": {(i,): [pulse.DriveChannel(i)] for i in range(self.num_qubits)},
+            "measure": {(i,): [pulse.MeasureChannel(i)] for i in range(self.num_qubits)},
+            "control": {
+                (edge): [pulse.ControlChannel(i)] for i, edge in enumerate(self._coupling_map)
+            },
+        }
+        setattr(self, "channels_map", channels_map)
+
+    def _get_noise_defaults(self, name: str) -> tuple:
+        """Return noise default values/ranges for duration and error of supported
+        instructions. There are two possible formats:
+            - (min_duration, max_duration, min_error, max_error),
+              if the defaults are ranges.
+            - (duration, error), if the defaults are fixed values.
+        """
+        return _NOISE_DEFAULTS.get(name, (1e-8, 9e-7, 1e-5, 5e-3))
```

Comment:
Can the default value for this be pulled out into a global so it is co-located with other data in this class? E.g. `_NOISE_DEFAULTS_FALLBACK`.